docs/app/usage/settings.rst

```diff
@@ -34,7 +34,7 @@ Enable Debug Logging
 Game Settings
 =============
 
-LOOT's game-specific settings can be customised by selecting a game in the sidebar. New game profiles can be added, making it easy to use LOOT across multiple copies of a game, including total conversion mods. LOOT ships with settings for the "Nehrim - At Fate's Edge" total conversion mod as an example. Game profiles can also be deleted, though the active game cannot have its profile deleted, and LOOT will recreate the profiles for the base games it supports (Morrowind, Oblivion, Skyrim, Skyrim Special Edition, Skyrim VR, Fallout 3, Fallout: New Vegas, Fallout 4 and Fallout 4 VR) when it is next run.
+LOOT's game-specific settings can be customised by selecting a game in the sidebar. New game profiles can be added, making it easy to use LOOT across multiple copies of a game, including total conversion mods. LOOT ships with settings for the "Nehrim - At Fate's Edge" total conversion mod as an example. Game profiles can also be deleted, though the active game cannot have its profile deleted, and LOOT will recreate the profiles for the base games it supports (Morrowind, Oblivion, Nehrim, Skyrim, Enderal, Skyrim Special Edition, Skyrim VR, Enderal Special Edition, Fallout 3, Fallout: New Vegas, Fallout 4 and Fallout 4 VR) when it is next run.
```

Comment:
Nehrim and Enderal aren't base game types, they use Oblivion, Skyrim or SkyrimSE as their base game types.

Reply:
@Ortham Ah, my bad, I'll drop the commit.